app/scripts/modules/openstack/serverGroup/configure/serverGroup.configure.openstack.module.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.serverGroup.configure', [
+  require('../../../core/account/account.module.js'),
+  require('./wizard/deployInitializer.controller.js'),
+  require('./ServerGroupCommandBuilder.js'),
+  require('./serverGroupConfiguration.service.js'),
+  require('./wizard/ServerGroupBasicSettings.controller.js'),
+  require('./wizard/Capacity.controller.js'),
+  require('./wizard/Clone.controller.js'),
+  require('./wizard/LoadBalancers.controller.js'),
+  require('./wizard/templateSelection.controller.js')
```

Comment:
This is consistent with Kubernetes, where @lwander inexplicably decided to start upper camel casing files and I inexplicably didn't say anything.

Lower camel case is preferred for file names. That said, getting git to accept file name changes where only the case changes can be a nightmare - I'm okay with this, with the caveat that someone might go in and change them (and k8s) to be consistent - very happy to accept a PR around that!